test/testing-utils.ts

```diff
@@ -111,10 +120,64 @@ export async function createTest<T>(
   try {
     await testFn({
       ...result,
+      fixturesDir,
       distDir,
       distFile,
     })
   } finally {
     await removeDirectory(distDir)
+
+    // Additional files to remove inside the fixtures directory
+    // e.g. tsconfig.json
+    if (filesToRemove) {
+      for (const file of filesToRemove) {
```

Comment:
Added `filesToRemove` to `createTest` to remove additional files created during tests (e.g. tsconfig.json)

Reply:
I see. Will remove it thanks!

Reply:
If you don't mind, could you tell me why you're not using temporary dirs?

Reply:
I want to do e2e at the begining, and linked bunchee there, so it can be seaprate from the installed packages in current project. but it's a long term refactor work